apis/apiextensions/v1/conditions.go

```diff
@@ -86,3 +89,12 @@ func TerminatingClaim() xpv1.Condition {
 		Reason:             ReasonTerminatingClaim,
 	}
 }
+
+// ReconcilePaused returns paused condition with human readable message
+// TODO: ReconcilePaused from crossplane-runtime does not set the message
```

Comment:
I would suggest just opening a PR there, but given the feature freeze deadline, can you at least open an issue to follow up?

Reply:
Actually, I just realized that I do not need to modify crossplane-runtime: on condition there is `WithMessage()` method so the human readable message can be provided easily.